catalog/plugin/catalog-plugin-pointofcontactupdate/src/test/java/org/codice/ddf/catalog/plugin/metacard/PointOfContactUpdatePluginTest.java

```diff
@@ -0,0 +1,193 @@
+        DeleteRequest deleteRequest = mock(DeleteRequest.class);
+        QueryRequest queryRequest = mock(QueryRequest.class);
+        ResourceRequest resourceRequest = mock(ResourceRequest.class);
+
+        DeleteResponse deleteResponse = mock(DeleteResponse.class);
+        QueryResponse queryResponse = mock(QueryResponse.class);
+        ResourceResponse resourceResponse = mock(ResourceResponse.class);
+
+        assertThat(pointOfContactUpdatePlugin.processPreCreate(createRequest), is(createRequest));
+        assertThat(pointOfContactUpdatePlugin.processPreDelete(deleteRequest), is(deleteRequest));
+        assertThat(pointOfContactUpdatePlugin.processPostDelete(deleteResponse),
+                is(deleteResponse));
+        assertThat(pointOfContactUpdatePlugin.processPreQuery(queryRequest), is(queryRequest));
+        assertThat(pointOfContactUpdatePlugin.processPostQuery(queryResponse), is(queryResponse));
+        assertThat(pointOfContactUpdatePlugin.processPreResource(resourceRequest),
+                is(resourceRequest));
+        assertThat(pointOfContactUpdatePlugin.processPostResource(resourceResponse,
+                mock(Metacard.class)), is(resourceResponse));
+
+        verifyZeroInteractions(createRequest,
+                deleteRequest,
+                queryRequest,
+                resourceRequest,
+                deleteResponse,
+                queryResponse,
+                resourceResponse);
+    }
+
+    private List<Metacard> getListOfUpdatedMetacards() {
+        Metacard resourceMetacard1 = getMetacardWithIdAndTag(RESOURCE_ID1, "resource");
+        Metacard resourceMetacard2 = getMetacardWithIdAndTag(RESOURCE_ID2, null);
+        Metacard registryMetacard = getMetacardWithIdAndTag(REGISTRY_ID, "registry");
+
+        return Arrays.asList(resourceMetacard1, resourceMetacard2, registryMetacard);
+    }
+
+    private Map<String, Metacard> getPreviousMetacardsWithPointOfContact() {
+        Attribute pocAttribute = new AttributeImpl(Metacard.POINT_OF_CONTACT, ADMIN_EMAIL);
+        Metacard resourceMetacard1 = getMetacardWithIdAndTag(RESOURCE_ID1, "resource");
+        resourceMetacard1.setAttribute(pocAttribute);
+        Metacard resourceMetacard2 = getMetacardWithIdAndTag(RESOURCE_ID2, null);
+        resourceMetacard2.setAttribute(pocAttribute);
+        Metacard registryMetacard = getMetacardWithIdAndTag(REGISTRY_ID, "registry");
+        registryMetacard.setAttribute(pocAttribute);
+
+        Map<String, Metacard> existingMetacards = new HashMap<String, Metacard>();
+        existingMetacards.put(RESOURCE_ID1, resourceMetacard1);
+        existingMetacards.put(RESOURCE_ID2, resourceMetacard2);
+        existingMetacards.put(REGISTRY_ID, registryMetacard);
+
+        return existingMetacards;
+    }
+
+    private Metacard getMetacardWithIdAndTag(String id, String tag) {
+        MetacardImpl metacard = new MetacardImpl();
+        metacard.setId(id);
+
+        if (tag != null) {
```

Comment:
I think someone mentioned to me once to use StringUtils.isEmpty() instead of using != null for Strings